apps/sms/test/marionette/lib/views/conversation/accessors.js

```diff
@@ -0,0 +1,49 @@
+var SELECTORS = Object.freeze({
+  main: '#thread-messages',
+  message: '.message .bubble',
+  headerTitle: '#messages-header-text',
+  container: '#messages-container',
+  callButton: '#messages-call-number-button'
+});
+
+module.exports = {
+  get message() {
+    return client.helper.waitForElement(SELECTORS.message);
+  },
+
+  get headerTitle() {
+    return client.helper.waitForElement(
+      SELECTORS.headerTitle
+    );
+  },
+
+  getMessageContent: function(message) {
+    return client.helper.waitForElement(
+      message.findElement(SELECTORS.content)
+    );
+  },
+
+  findMessage: function(id) {
+    return client.findElement('.message[data-message-id="' + id + '"]');
+  },
+
+  scrollUp: function() {
+    var conversationContainer = client.findElement(
+      SELECTORS.container
+    );
+
+    actions.flick(conversationContainer, 50, 50, 50, 350).perform();
+  },
+
+  get callButton() {
```

Comment:
Could you please move this to "properties" section above, so that first we define all properties, next methods.